src/components/dashboard/EntityStatOverview/useActiveEntityCount.ts

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@supabase-cache-helpers/postgrest-swr';
+import { getLiveSpecShards } from 'api/liveSpecsExt';
+import { isArray } from 'lodash';
+import { useMemo } from 'react';
+import { useTenantStore } from 'stores/Tenant/Store';
+import { Entity } from 'types';
+import { hasLength } from 'utils/misc-utils';
+
+export default function useActiveEntityCount(entityType: Entity) {
+    const selectedTenant = useTenantStore((state) => state.selectedTenant);
+
+    const { data, error, isValidating } = useQuery(
+        hasLength(selectedTenant)
+            ? getLiveSpecShards(selectedTenant, entityType)
+            : null,
+        {
+            refreshInterval: 15000,
+        }
+    );
+
+    const count = useMemo(
+        () =>
+            data ? data.filter(({ disable }) => disable !== true).length : 0,
+        [data]
+    );
```

Comment:
I understand why we are calculating `count` but again might end up causing some confusion since it isn't the normal count that get's returned by queries.

Reply:
From what I have read, there is not a way to reference the value of or extract an object property in a query formed using the Supabase JavaScript client. Would you like this variable renamed?

Reply:
No... we can worry about it if it actually causes a problem during future development.